tests/execution/graphql-conformance-http-007.md

```diff
@@ -0,0 +1,84 @@
+---
+skip: true
+---
+
+# Test named fragments. Skipped because Tailcall does not construct correctly the interface types based on \_\_typename field.
+
+```graphql @config
+schema
+  @server(port: 8001, queryValidation: false, hostname: "0.0.0.0")
+  @upstream(baseURL: "http://upstream/", httpCache: 42) {
+  query: Query
+}
+
+type Query {
+  profiles(handles: [ID!]!): [Profile!]! @http(path: "/profiles", query: [{key: "handles", value: "{{.args.handles}}"}])
+}
+
+interface Profile {
+  id: ID!
+  handle: String!
+}
+
+type User implements Profile {
+  id: ID!
+  handle: String!
+  friends: Counter!
+}
+
+type Page implements Profile {
+  id: ID!
+  handle: String!
+  likers: Counter!
+}
+
+type Counter {
+  count: Int!
+}
+```
+
+```yml @mock
+- request:
+    method: GET
+    url: http://upstream/profiles?handles=user-1&handles=user-2
+  expectedHits: 1
+  response:
+    status: 200
+    body:
+      - id: 1
+        handle: user-1
+        __typename: User
```

Comment:
upstream won't send `__typename` in most cases, especially rest upstream

Reply:
could you also check other tests for this? instead of relying on `__typename` from the upstream Tailcall should be able to figure the type on it's own

Reply:
I just removed the __typename. This test can test this behavior